app/migrations/versions/f8ce3abf47fb_.py

```diff
@@ -0,0 +1,39 @@
+"""empty message
```

Comment:
don't have an opinion on this- but would it make sense to merge all three migrations into one?